### DISCUSSION TITLE
Feature idea: Automatic paging detection like bat's

### DISCUSSION THREAD

**Discussion by dievilz** [noborus/ov#432] [Ideas]
I use bat with ov, but it seems that ov is not respecting bat's automatic paging feature.
I'm fairly certain I didn't misconfigured something in my dotfiles (like using an alias like bat --paging=always or something), so I want to know if you guys have a clue of what could be happening, and, if the integration of that functionality itself in ov would be feasible or convenient (in case ov doesn't have it yet)?

**Comment by noborus** [comment]
I think automatic paging of bat is realized by pager option.
https://github.com/sharkdp/bat#automatic-paging

I think it comes with the `--quit-if-one-screen` option for `less` by default.
Therefore, you should start `ov` with the `--quit-if-one-screen` or `-F` option.

```
bat --paging=auto --pager="ov --quit-if-one-screen" a
File: a
1
2
3
```
